README.md

```diff
@@ -121,31 +127,68 @@ And start or stop a specific service with:
     $ pebble start <name1> [<name2> ...]
     $ pebble stop  <name1> [<name2> ...]
 
+## API
+
+The documentation for the currently exposed API endpoints can be found in `doc/api.yaml`
```

Comment:
Hmm, I haven't been updating this, so it's already out of date, and it requires Node/npx to update. I wonder if we should just drop it for now. Maybe we can add again later once most of the Pebble API and features are in place. In any case, it probably won't be used by anyone except Pebble devs, in which case we can just say "refer to the code". :-) Everyone else will use the Pebble Python client in Operator Framework, or the Pebble Go client or CLI.

Reply:
I don't disagree, will remove for now.